.aws/deploy/backend-task-definition.prod.json

```diff
@@ -0,0 +1,221 @@
+{
+  "containerDefinitions": [
+    {
+      "name": "backend",
+      "portMappings": [
+        {
+          "containerPort": 8081,
+          "hostPort": 8081,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "environment": [],
+      "mountPoints": [
+        {
+          "sourceVolume": "efs",
```

Comment:
nit: is this our own alias? if yes could we be more descriptive? `repos-efs-1` or smt like that?

Reply:
hmm i think the source directory is `/efs` and we're mounting that directory (cos we mount efs) not the `/efs/repos` directory, so i'll probably be keeping this?

i could change this to `ggs-efs` if that feels more descriptive

Reply:
oki